src/test/ui/const-generics/min_const_generics/forbid-non-static-lifetimes.full.stderr

```diff
@@ -0,0 +1,15 @@
+error: internal compiler error: compiler/rustc_mir/src/borrow_check/universal_regions.rs:LL:CC: cannot convert `ReFree(DefId(0:12 ~ forbid_non_static_lifetimes[317d]::issue_73375[0]), BrNamed(DefId(0:13 ~ forbid_non_static_lifetimes[317d]::issue_73375[0]::'a[0]), 'a))` to a region vid
```

Comment:
Why are we adding a unit test for an ICE?

Reply:
I added this [when it was requested by a review comment](https://github.com/rust-lang/rust/pull/76739#discussion_r488631207), I can remove it if you'd prefer that.

Reply:
I didn't know it'd ICE when I added the comment -- I'd add a comment saying why the ICE happens and a link to the issue, and why we cannot test with revisions.

Reply:
I've removed the ICE test and slightly updated the comment.